backend/python/app/services/implementations/story_service.py

```diff
@@ -245,3 +249,6 @@ def get_story_translations_available_for_review(self, language, level):
         except Exception as error:
             self.logger.error(str(error))
             raise error
+
+    def _get_num_translated_lines(self, translation_contents):
+        return sum(1 for _ in translation_contents if _.translation_content)
```

Comment:
How does this handle empty strings?

Reply:
i believe Python empty strings evaluate to False so it should only sum if the translation_content is not an empty string--this seemed to be the case when i was testing but let me know if it's not working for you/if there's a more elegant way to do the check i'm happy to change it!!